src/Tizen.Sensor/Tizen.Sensor/EventArgs/RecorderDataEventArgs.cs

```diff
@@ -0,0 +1,45 @@
+﻿/*
+* Copyright (c) 2019 Samsung Electronics Co., Ltd. All Rights Reserved
+*
+* PROPRIETARY/CONFIDENTIAL
```

Comment:
Why this file has different Copyright comment header than the rest of the committed files. What makes it confidential?

Reply:
It doesn't have any confidential data. Copied wrong header file in RecorderDataEventArgs.cs and  ProviderIntervalChangedEventArgs.cs.   Will change the header files in next commit.